proto/src/determined/api/v1/model.proto

```diff
@@ -116,6 +116,8 @@ message PostModelRequest {
   google.protobuf.Struct metadata = 3;
   // Labels associated with this model.
   repeated string labels = 4;
+  // The id of the workspace associated with this model.
+  int32 workspace_id = 5;
```

Comment:
make it `optional` (and then downstream check for it being `nil` instead of somewhat ambiguous 0).